packages/api/src/external/carequality/document/process-outbound-document-retrieval-resps.ts

```diff
@@ -204,13 +277,113 @@ export async function processOutboundDocumentRetrievalResps({
   }
 }
 
+async function handleOperationOutcomeIssues({
+  issues,
+  requestId,
+  patientId,
+  cxId,
+}: {
+  issues: Issue[];
+  requestId: string;
+  patientId: string;
+  cxId: string;
+}) {
+  const { log } = out(
+    `CQ handleOperationOutcomeIssues - requestId ${requestId}, patient ${patientId}`
+  );
+  const existingFHIRDocRefs: DocumentReference[] = await getDocumentsFromFHIR({
+    cxId,
+    patientId,
+    documentIds: issues.map(issue => issue?.id ?? ""),
+  });
+
+  const transactionBundle: Bundle = {
+    resourceType: "Bundle",
+    type: "transaction",
+    entry: [],
+  };
+
+  for (const docRef of existingFHIRDocRefs) {
+    if (!docRef.id) continue;
+
+    const updatedFHIRDocRef: DocumentReferenceWithId = {
+      ...docRef,
+      id: docRef.id,
+      status: "entered-in-error",
```

Comment:
this is how I decided to denote documents we get no response or an error response for